app/assets/stylesheets/schools/dashboard.scss

```
@@ -7,6 +7,23 @@
     .govuk-summary-list__actions {
       width: 13%;
     }
+    .govuk-summary-list__missing {
+      border-left: 10px solid #1d70b8;
+      padding-left: 15px;
+    }
+    .govuk-summary-list__missing-heading {
+      margin-bottom: 10px;
+    }
+  }
+
+  .filter {
```

Could we start using the `app-` prefix for our own components? That’s [recommended by GDS](https://design-system.service.gov.uk/get-started/extending-and-modifying-components/) to avoid naming conflicts. I don't always remember but I'll try to!

```suggestion
  .app-filter {
```